New navigation (#4614)
timescale/docs: use-timescale/services/service-overview.md

    path:
        - [services, :serviceId, overview]
---

# $CONSOLE_LONG overview

When you log into [$CONSOLE][cloud-login], you see the $PROJECT_SHORT overview. 

![Check $SERVICE_LONG is running](https://assets.timescale.com/docs/images/tiger-on-azure/tiger-console-services-view.png)

The project view includes the following tabs:

> - `Services`: see all $SERVICE_SHORTs created in your $PROJECT_SHORT and their statuses. See [Create a $SERVICE_LONG][create-service].

Reviewer: Not too much of a fan of See links, please add the link into the sentence for all these items.

- `Security`:  create $VPCs and IP allowlists. See [Security][security].
- `Exporters`: export metrics and logs from your $SERVICE_SHORTs. See [Metrics and logging][metrics-logging]. 
- `Settings`: create client credentials, add AI model API keys, and request a SOC 2 report. See [Create client credentials][find-connection-info].
- `Users`: add and remove users in your $PROJECT_SHORT. See [Control user access to Tiger Cloud projects][members].
- `Billing`: check usage, change $PRICING_PLANs, and manage payment methods. See [Pricing plans and account management][pricing].

> - `Data view`: write queries with autocomplete, save them in folders, share them, and create charts/dashboards with the added power of AI. See [Data view][run-queries-data-view].

Reviewer: ```suggestion
- `Data view`: write queries with autocomplete for any $SERVICE_SHORT, save them in folders, share them, and create charts/dashboards with the added power of AI. See [$DATA_MODE_CAP][run-queries-data-view].
```

Can you create a PR in web-documentation to update the vars please.

> Click a $SERVICE_SHORT to manage and configure it. Each $SERVICE_SHORT hosts a single database managed for you by $CLOUD_LONG.

Reviewer: ```suggestion
To manage and configure a $SERVICE_LONG, click it. Each $SERVICE_SHORT is an optimized [$PG][postgres-link] instance extended with database engine innovations such as [$TIMESCALE_DB][timescaledb-link], managed for you by $CLOUD_LONG.
```

[postgres-link]: /api/:currentVersion:/glossary/#postgresql
[timescaledb-link]: /api/:currentVersion:/glossary/#timescaledb

If you need more than one database, [create a new $SERVICE_SHORT][create-service].

![Select a query to edit](https://assets.timescale.com/docs/images/tiger-on-azure/ops-mode-overview-tiger-console.png)

The individual $SERVICE_SHORT view includes the following tabs:

- `Overview`: get an overview of your $SERVICE_SHORT details, infrastructure, [connection info][connect-service], and [performance optimization tips][recommendations].

> - `Actions`: connect and import data into your $SERVICE_SHORT. See [Import and sync][import-sync].

Reviewer: Not too much of a fan of See links, please add the link into the sentence. 
`Actions`: [connect and import data][import-sync] into your $SERVICE_SHORT.

Same for the others.

- `Explorer`: inspect objects and storage configuration for your $SERVICE_SHORT. See [$SERVICE_LONG explorer][service-explorer].
- `Monitoring`: get visibility into your $SERVICE_SHORT performance. See [Monitor your $SERVICE_LONGs][monitoring].
- `Connectors`: sync or stream directly, so data from another source is continuously updated in your $SERVICE_SHORT. See [Import and sync][import-sync].
- `SQL editor`: query your $SERVICE_SHORTs. See [$SQL_EDITOR][run-queries-sql-editor].
- `Operations`: fully manage your $SERVICE_SHORT: [configure compute and storage][change-resources], [set up replicas][replication], [back up][backup], [upgrade][upgrade], and more. 

> 

Reviewer: Can you add the AI tab please.

    path:
        - [services, :serviceId, overview]
---

# $CONSOLE_LONG overview

> When you log into [$CONSOLE][cloud-login], you see the $PROJECT_SHORT overview. 

Reviewer: Can you turn this into some kind of Bla Bla. 

You use [$CONSOLE][cloud-login] to solve world peace, make the best tea and travel through time ...

![Check $SERVICE_LONG is running](https://assets.timescale.com/docs/images/tiger-on-azure/tiger-console-services-view.png)

> You see the following tabs:

Reviewer: ```suggestion
When you log into [Tiger Cloud Console](https://console.cloud.timescale.com/), you see the following tabs:
```

> - `Services`: [see all $SERVICE_SHORTs][create-service] created in your $PROJECT_SHORT and their statuses. Each $SERVICE_SHORT is an optimized [$PG][postgres-link] instance extended with database engine innovations such as [$TIMESCALE_DB][timescaledb-link], managed for you by $CLOUD_LONG. If you need more than one database, [create a new $SERVICE_SHORT][create-service].

Reviewer: ```suggestion
- `Services`: create a new [$SERVICE_SHORT][create-service] in your $PROJECT_SHORT, and list the existing $SERVICE_SHORT and their statuses. Each $SERVICE_SHORT is an optimized [$PG][postgres-link] instance extended with database engine innovations such as [$TIMESCALE_DB][timescaledb-link], managed for you by $CLOUD_LONG.
```

- `Security`:  create [$VPCs][vpc] and [IP allowlists][ip-allowlist]. 
- `Exporters`: [export metrics and logs][metrics-logging] from your $SERVICE_SHORTs. 
- `Activity`: [check your $SERVICE_SHORTs][activity-log] and $PROJECT_SHORT activity. 
- `Settings`: [create client credentials][find-connection-info], add AI model API keys, and request a SOC 2 report. 
- `Users`: [add and remove users][members] in your $PROJECT_SHORT. 
- `Billing`: [check usage][check-usage], [change $PRICING_PLANs][pricing], and manage payment methods. 
- `Data view`: write [queries with autocomplete][run-queries-data-view] for any $SERVICE_SHORT, save them in folders, share them, and create charts/dashboards with the added power of AI. 

> To manage and configure a $SERVICE_LONG, click it. 

Reviewer: Can you change this to something along the lines of:

When you select a $SERVICE_LONG, you see the $SERVICE_SHORT `Overview`:
